indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -323,3 +326,27 @@ class RHRegistrationsExportCSV(RHRegistrationsExportBase):
     def _process(self):
         csv_file = generate_csv_from_registrations(self.registrations, self.regform_items, self.special_items)
         return send_file('registrations.csv', csv_file, 'text/csv')
+
+
+class RHRegistrationEditPayment(RHManageRegistrationBase):
+    """Modify the payment status of a registration"""
+
+    def _checkParams(self, params):
+        RHManageRegistrationBase._checkParams(self, params)
+        self.payment_status = request.form.get('payment_status')
```

Comment:
`request.form['payment_status'] == '1'` (and I think you could safely do it in `_process` instead of using checkParams for it - it's not something where doing it here adds any benefits)